src/plugins/chart_expressions/expression_gauge/public/components/utils/gauge_types.ts

```diff
@@ -6,15 +6,16 @@
  * Side Public License, v 1.
  */
 
-import { GoalProps } from '@elastic/charts';
+import { BulletProps, BulletSubtype } from '@elastic/charts';
 import { GaugeShape, GaugeShapes } from '../../../common';
 
-export const getSubtypeByGaugeType = (type: GaugeShape): GoalProps['subtype'] =>
+export const getSubtypeByGaugeType = (type: GaugeShape): BulletProps['subtype'] =>
```

Comment:
more of a design question @MichaelMarcialis  and shouldn't stop from merging this PR:
Now that we have so many types in this group, maybe it's better to divide it in two groups - metric ones and the goal/gauge ones? 
<img width="425" alt="Screenshot 2024-03-13 at 11 47 40" src="https://github.com/elastic/kibana/assets/4283304/8d6b454b-d52c-403c-a95c-fc3458f98580">

Reply:
Yup, I had discussed this with @markov00 and @gvnmagni to group them in a simpler less verbose way. But we decided to push this off until the chart switcher was updated at which time we could simplify the xy groupings too.

Something like this...

https://github.com/elastic/kibana/assets/19007109/17fbeb0e-dbd7-4e95-879f-c8567c346303

Reply:
I've created an issue here to track this: https://github.com/elastic/kibana/issues/179260 Please continue the conversation here